packages/marshal/src/rankOrder.js

```diff
@@ -292,19 +293,41 @@ export const makeComparatorKit = (compareRemotables = (_x, _y) => NaN) => {
 
   return harden({ comparator: outerComparator, antiComparator });
 };
+harden(makeComparatorKit);
+
 /**
  * @param {RankCompare} comparator
  * @returns {RankCompare=}
  */
 export const comparatorMirrorImage = comparator =>
   comparatorMirrorImages.get(comparator);
+harden(comparatorMirrorImage);
+
+export const { comparator: compareRank, antiComparator: compareAntiRank } =
+  makeComparatorKit();
```

Comment:
Many drive by's to this file. Could be broken off into a separate PR is reviewers request.